test/battle/weather/snow.c

```diff
@@ -30,18 +30,20 @@ SINGLE_BATTLE_TEST("Snow multiplies the defense of Ice-types by 1.5x", s16 damag
 
 SINGLE_BATTLE_TEST("Snowscape fails if Desolate Land is active")
```

Comment:
Please update the name of the test to account for the change.

Reply:
done